pyLDAvis/js/ldavis.js

```diff
@@ -4,39 +4,83 @@
 
 'use strict';
 
-var LDAvis = function(to_select, data_or_file_name) {
+//var LDAvis = function(to_select, data_or_file_name_a, data_or_file_name_b) {
+var LDAvis = function(to_select, data_or_file_name_list) {
+    console.log(to_select);
 
     // This section sets up the logic for event handling
     var current_clicked = {
         what: "nothing",
         element: undefined
     },
-        current_hover = {
-            what: "nothing",
-            element: undefined
-        },
-        old_winning_state = {
-            what: "nothing",
-            element: undefined
-        },
-        vis_state = {
-            lambda: 1,
-            topic: 0,
-            term: ""
-        };
+    current_hover = {
+        what: "nothing",
+        element: undefined
+    },
+    old_winning_state = {
+        what: "nothing",
+        element: undefined
+    },
+    vis_state = {
+        lambda: 1,
+        topic: 0,
+        term: ""
+    };
+
+    var current_clicked2 = {
+        what: "nothing",
+        element: undefined
+    },
+    current_hover2 = {
+        what: "nothing",
+        element: undefined
+    },
```

Comment:
Yes, using two variables shouldn't be used. I was trying to see how to reuse visualise() function for more than one model, because the way D3 code is written will work for one model only. To make support for multi model, lot of elements in svg either need to be made global, or, (better approach) associate each model with separate ids, and store K, R etc. as arrays or other data structure. 

However, I am facing problem is sending ids from `prepared_data_to_html()` in `_display.py`. I am glad you took up this issue. Thanks a lot.